java/kotlin-extractor/BUILD.bazel

```diff
@@ -178,8 +178,18 @@ kt_javac_options(
         ) for variant in ("standalone", "embeddable") for version in VERSIONS],
         visibility = ["//visibility:public"],
     ),
-    sh_binary(
-        name = "print-default-version",
-        srcs = ["//java/kotlin-extractor/defaults:default-version-printer"],
+    genrule(
+        name = "versions-list",
+        outs = ["kotlin-versions.list"],
+        cmd = "\n".join(["cat > $@ << EOF"] + VERSIONS + ["EOF"]),
```

Comment:
Did we test this on Windows?

Reply:
I know it does, but due to msys paths sometimes genrules on Windows still behave weirdly... better check to be safe!